keras_cv/layers/preprocessing/auto_contrast_test.py

```diff
@@ -90,3 +156,11 @@ def test_auto_contrast_properly_converts_value_range(self):
 
         self.assertTrue(tf.math.reduce_any(ys[0] == 0.0))
         self.assertTrue(tf.math.reduce_any(ys[0] == 1.0))
+
+    def test_is_consistent_with_non_vectorized_implementation(self):
```

Comment:
@sebastian-sz can you please move this test case to the benchmark, instead of this test case?

Then we can merge -- thanks!

Reply:
@ianstenbit done.
1. Moved consistency test to benchmarks.
2. Added `get_random_transformation_batch` override as it's unused.